articles/protocols/oauth2/oauth-implicit-protocol.md

```diff
@@ -30,9 +30,7 @@ The steps are quite simple though:
         &state=VALUE_THAT_SURVIVES_REDIRECTS</code></pre>
 
 
-  Your code would then parse the hash segment of the URL and extract the parameters: `access_token` and `id_token`.
-
-  > The `access_token` can then be used to call Auth0's `userinfo` endpoint to get the attributes of the user.
+  Your code would then parse the hash segment of the URL and extract the parameters: `access_token` and `id_token`. The `access_token` can then be used to call Auth0's `userinfo` endpoint to get the attributes of the user.
```

Comment:
@cabralmartin The [endpoint](https://auth0.com/docs/api/authentication#!#get--userinfo) mentions access_token but I want to confirm that this won't change with p2.